tests/TestHarness/Cluster.py

```diff
@@ -920,7 +946,7 @@ def createAccountAndVerify(self, account, creator, stakedDeposit=1000, stakeNet=
     #         return transId
     #     return None
 
-    def createInitializeAccount(self, account, creatorAccount, stakedDeposit=1000, waitForTransBlock=False, stakeNet=100, stakeCPU=100, buyRAM=10000, exitOnError=False):
+    def createInitializeAccount(self, account, creatorAccount, stakedDeposit=1000, waitForTransBlock=False, stakeNet=100, stakeCPU=100, buyRAM=10000, exitOnError=False, sign=False, additionalArgs=''):
```

Comment:
We've added `sign` and `additionalArgs` but I don't see where we are using them. If this is future expansion, that's okay, but we ought to signal that intent with some documentation. Otherwise it should be removed.